### PR TITLE
Added arguments

### PR DESCRIPTION
**Pull request by theDanielJLewis**
Added optional arguments for email, password, platform, and games.

It would be nice to make `--games` support a list of games to easily process some or all of them on a single platform. For example, `--games=1,tps,3`.

### CODE REVIEW

File: src/bin/prompts.js [hunk 1]
```diff
@@ -1,51 +1,78 @@
...
-      name: 'email',
-      message: 'Shift Email'
-    },
-    {
-      type: 'password',
-      name: 'password',
-      message: 'Shift Password'
-    }
-  ]);
+  if ( argv['email'] && argv['password'] ) {
```
[the remark this answers is not in the document]
Comment (trs) [reply src/bin/prompts.js]: Or even pass in the argument value, and only prompt if it is `undefined`

```js
// prompts.js
async function promptEmail(email) {
  if (email !== undefined) return email;
  ({email} = await prompt({
     ...
  }));
  return email;
}
```

```js
// index.js
const email = await promptEmail(argv.email);
```

File: src/bin/prompts.js [hunk 2]
```diff
@@ -1,51 +1,78 @@
...
+      {
+        type: 'select',
+        name: 'platform',
+        message: 'Platform',
+        choices: PLATFORM_NAMES.map((title, value) => ({title, value}))
+      }
+    ]);
+  }
+  
+  if ( argv['games'] ) {
```
Comment (trs) [inline src/bin/prompts.js]: Can do the same split up with this function, `promptGame` `promptPlatform`

File: README.md [hunk 3]
```diff
@@ -29,10 +29,27 @@ Automatically redeem Borderlands Shift codes from the wonderful website http://t
...
+    `1`: Borderlands: Game of the Year
+    `2`: Borderlands 2
+    `tps`: Borderlands: The Pre-Sequel
+    `3`: Borderlands 3
+
 ## FAQ
 
 1. What does `"please launch a shift-enabled title first"` mean?
-    - You can only redeem a certain number of SHiFT codes before you'll see this. It means you need to open a SHiFT enabled title (a Borderlands game) and play past the main menu. Once you're loaded in, you can exit the game and continue redeeming.
+    - You can only redeem a certain number of SHiFT codes within time window before you'll see this. It means you need to open a SHiFT enabled title (a Borderlands game) and play past the main menu. Once you're loaded in, you can exit the game and continue redeeming. You may also wait at least an hour and try again.
```
[the remark this answers is not in the document]
Comment (theDanielJLewis) [reply README.md]: Loading up the game is an instant resolution, yes. But I also found that if I try again at least an hour later, it works. I'm sorry I haven't timed it specifically.

File: src/bin/prompts.js [hunk 4]
```diff
@@ -1,51 +1,78 @@
...
-      message: 'Shift Password'
-    }
-  ]);
+  if ( argv['email'] && argv['password'] ) {
+      var email = argv['email'];
+      var password = argv['password']; 
+  } else {
+    statusLog.success('Login to your SHiFT account:');
+
+    var {email, password} = await prompt([
```
[the remark this answers is not in the document]
Comment (theDanielJLewis) [reply src/bin/prompts.js]: Sounds good!

File: src/bin/prompts.js (hunk 2 above)
Comment (theDanielJLewis) [reply src/bin/prompts.js]: Yeah, that would probably be better than my code.

**Comment by theDanielJLewis** [comment]
I'm not extremely experienced with Git, so handle this pull request however you need to.

**Comment by trs** [comment]
@theDanielJLewis are you wanting to make those changes? If not, I can merge this and make them in this branch.

**Comment by theDanielJLewis** [comment]
I’ll let you handle that. I might not be able to try anything for a while still.

**Comment by trs** [comment]
Great work man! Much appreciated

**Pull request merged**